### PR TITLE
Add MD builder short answer

### PR DESCRIPTION
**Pull request by ChinemeremChigbo**
## Notion ticket link

[[Module Builder] Short Answer](https://www.notion.so/uwblueprintexecs/Module-Builder-Short-Answer-1a1a3f4784d84ea29f40e78a1952a720)

## Implementation description
* Made Short Answer Module Builder Section

## Steps to test
1. Add short answer module builder section
2. Play around with values and ensure no new bugs are discovered
3. Look through the changed files on github and ensure no major coding qualms are present

## What should reviewers focus on?
* Module builder short answer section

image

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by chamod-gamage** [comment]
@ChinemeremChigbo Great work here! The component looks super clean
From a product perspective, we don't necessarily need this to be a builder (since admin can add sections of the same type) but since you've built it out that way we can go ahead with it that way anyways. One thing though with the builder is we need to allow admin to delete a given question (I'd suggest just putting an X somewhere) so that they can have a Short Answer component with just one question + textbox.
Code looks well written from my skim but I'll let other reviewers do a deeper dive.
Should be a quick change and then you'll be good to merge!

**Review by chamod-gamage — CHANGES_REQUESTED** [review]
(See above)
Let's add a (X) for each question so that admin may delete

**Comment by ChinemeremChigbo** [comment]
Do we plan to implement deletion of short answers, multiple choice, multiselect, and entire sections? If so, we should probably add a notion ticket for this (maybe also include the deletion of entire modules just so all module deletion related tasks can be done in a fell swoop.

**Comment by chamod-gamage** [comment]
> Do we plan to implement deletion of short answers, multiple choice, multiselect, and entire sections? If so, we should probably add a notion ticket for this (maybe also include the deletion of entire modules just so all module deletion related tasks can be done in a fell swoop.

Yessir thanks for the reminder! I'd say make the individual Short Answer delete button in this PR and the rest can be its own ticket :)

**Review by chamod-gamage — APPROVED** [review]
Looks amazing! Thanks for all your hard work!

File: src/components/shortAnswer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    const deleteQuestion = (index: number) => {
+        const newQuestions = [...questions];
+        newQuestions.splice(index, 1);
+        setQuestions(newQuestions);
+    };
+
+    const questionFields = (
+        <Box>
+            {questions.map((currentQuestion, i) => (
```
Comment (tawnyzhao) [inline src/components/shortAnswer/index.tsx:74]: I think it will make it easier to support user variables if each content section only had 1 user input

**Review by tawnyzhao — CHANGES_REQUESTED** [review]

File: src/components/shortAnswer/index.tsx (hunk 1 above)
Comment (chamod-gamage) [reply src/components/shortAnswer/index.tsx:50]: Ah that's true, good product thinking

**Review by tawnyzhao — APPROVED** [review]

**Pull request merged**